de.peeeq.wurstscript/src/main/java/de/peeeq/wurstscript/attributes/prettyPrint/Main.java

```diff
@@ -0,0 +1,139 @@
+package de.peeeq.wurstscript.attributes.prettyPrint;
+
+import de.peeeq.wurstscript.gui.WurstGui;
+import de.peeeq.wurstscript.gui.WurstGuiCliImpl;
+import de.peeeq.wurstio.WurstCompilerJassImpl;
+import de.peeeq.wurstscript.RunArgs;
+import de.peeeq.wurstscript.ast.*;
+import de.peeeq.wurstscript.ast.CompilationUnit;
+import de.peeeq.wurstscript.gui.WurstGui;
+import de.peeeq.wurstscript.gui.WurstGuiCliImpl;
+
+import org.eclipse.jdt.annotation.Nullable;
+import java.nio.charset.StandardCharsets;
+import java.io.StringReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import org.apache.commons.lang.StringUtils;
+
+import static de.peeeq.wurstscript.utils.Utils.printElement;
+
+
+public class Main {
+
+    /**
+     * @param args
+     */
+    public static void main(String[] args) {
```

Comment:
shouldn't have it's own main func.
Instead have runarg for external use and WorkerTask for vscode integration